doc/features.md

```diff
@@ -1,44 +1,28 @@
 # Optional Features
 
-These features are not enabled by default. See each section below on how
-to enable the feature if desired.
+These features require CGI scripts to execute on your server. In order for them
+to function, make sure your server is configured to allow the execution of
+CGI files. You can then enable these features in your configuration file;
 
 ## Data Download
 
-The data download feature requires a CGI script to execute queries to CMR
-on behalf of Worldview. For this reason, it is not enabled by default.
-If you understand the implications of this feature and wish to enable it,
-edit the ``options/features.json`` file and change to:
+This feature uses CGI scripts to query the CMR API on the server. To enable,
+open `options/common/features.json` and set `"dataDownload": true`.
 
-    "dataDownload": true
+## URL Shortening
 
-Rebuild Worldview and the CGI script should be included in the
-``service`` directory. If your system administrator has disabled
-``.htaccess`` files, configure apache to allow the execution of this
-CGI script.
+This feature uses
+[bit.ly](http://bit.ly) to shorten links. Follow these steps to enable it;
 
-## URL Shortening
+* Edit `options/common/features.json` and set `"urlShortening": true`.
+* Obtain a login and API key from [bit.ly](http://bit.ly).
+* Create `build/options/bitly.json` with the following contents (replacing `your_login` and `your_key` with the appropriate values);
+
+  ```json
+  {
+    "login": "your_login",
+    "key": "your_key"
+  }
+  ```
 
-The URL shortening feature uses the service provided by
-[bit.ly](http://bit.ly). You will need to obtain an API key to use this
-service. Once you have obtained a key, create a ``options/bitly.json``
-file using the following as a template:
-
-```json
-{
-    "login": "my_login",
-    "key": "xh367ahdfjwna"
-}
-```
-
-This feature also requires a CGI script to execute queries to bit.ly
-on behalf of Worldview. For this reason, it is not enabled by default.
-If you understand the implications of this feature and wish to enable it,
-edit the ``options/features.json`` file and change to:
-
-    "urlShortening": true
-
-Rebuild Worldview and the CGI script should be included in the
-``service`` directory. If your system administrator has disabled
-``.htaccess`` files, configure apache to allow the execution of this
-CGI script.
+  > Caution: Do not commit this file to a public repo, and make sure that the `build/options` directory is not publicly accessible on your web server since your API key should remain secret.
```

Comment:
Good note to add here. How about, in the future, we create a githook which checks this directory and prevents committing code if there is sensitive information here. `pre-commit` hooks are normally something an individual has to setup on their machine but maybe we could look into using something like https://github.com/observing/pre-commit, to run an npm script on `git commit`, thoughts?

Reply:
How would we detect sensitive information?

From our side, I don't think we need an automatic check for this, since we've already got safeguards in place. We know better than to commit sensitive information. Even if we commit it locally, it's only a problem if that commit gets shared publicly, so we have time to catch it. And if it gets shared publicly we'll catch it during our code review process. In the event that a secret key got shared, we can always invalidate the key and generate a new one.

That being said, it could be a helpful safeguard to add for our users, if there's an easy way to implement detection of sensitive information.

Reply:
Your last paragraph sums up exactly what I was thinking; it'd be a nice catch for end-users if it is something we can easily add. Otherwise, there is no point in jumping through a bunch of hoops since WV doesn't contain any other sensitive data.

My idea would be to search for strings that would indicate private information.
See: https://github.com/jonDowdle/git-hooks/blob/master/pre-commit/reject-private-keys.sh

Here are some pre-built hooks using the pre-commit framework I linked above:
https://github.com/pre-commit/pre-commit-hooks